WordsLive.Core/Resources/thirdparty/jquery.js

```diff
@@ -1,2 +1,10716 @@
\ No newline at end of file
+/*!
```

Comment:
jquery.js was in a minified version before ... it probably doesn't make much of a difference, but minified can (in theory) be loaded/parsed faster. Was it an intentional decision to include it non-minified?
I know the other JS resources are also not minified, but maybe we should change that.

Reply:
I thought the loading time probably doesn't matter much because it is all local (without actually checking) and just used the files from the official package without any postprocessing. For jquery there is a minified file in the official package, but for pdfjs there isn't.
If I use the minified files for jquery and jquery-mousewheel, I would also switch the filename to how it is in the official package so that it is really clear what the source is.

Reply:
Okay, I checked the sizes and 85 KB (minified) vs 278 KB won't really matter seeing how big WordsLive already is.
Having it unminified might help when JS debugging is required.